src/bin/sage-runtests

```diff
@@ -66,6 +69,7 @@ if __name__ == "__main__":
     parser.add_argument("-i", "--initial", action="store_true", default=False, help="only show the first failure in each file")
     parser.add_argument("--exitfirst", action="store_true", default=False, help="end the test run immediately after the first failure or unexpected exception")
     parser.add_argument("--force_lib", "--force-lib", action="store_true", default=False, help="do not import anything from the tested file(s)")
+    parser.add_argument("--only-lib", action="store_true", default=False, help="skip Python/Cython files that are not installed as modules")
     parser.add_argument("--abspath", action="store_true", default=False, help="print absolute paths rather than relative paths")
```

Comment:
I understood the meaning of `--only-lib` by the statement "Use the option --only-lib to skip the source files of all Python/Cython modules that are not installed in the virtual environment..." in the documentation. I still don't understand the meaning of the existing `--force-lib`. The common `-lib` refers to the same thing, "the library installed to a virtual environment"?

Reply:
Well, in both cases, `lib` refers to the Sage library. 

When `sage -t` is applied to a Python file outside of the Sage library (= the installed copy or "the" source tree; technically, a file that does not look like it belongs to a source tree with the top-level package "sage"), then it first calls `load` on that file, which adds stuff to the global namespace. This is a feature that simplifies testing user code, as one does not have to say `from .... import` in every doctest. `--force-lib` turns this mechanism off and treats that file like any file that is part of the Sage library.

The `lib` in `--only-lib` is a bit different. It also checks whether the given file looks like it belongs to a source tree with the top-level package "sage" and then checks whether that is installed as a module in the installed Sage library.

Reply:
Thanks. Then how about `--only-installed` instead of `--only-lib` to avoid possible confusion with the `lib` in `--force-lib`? Perhaps together with `--all-installed` instead of `--installed`.

Reply:
I don't like `--only-installed` as a replacement for `--only-lib` because `--installed` runs doctests as they appear in the installed files, whereas `--only-lib` runs the doctests in the given files and only filters by an additional condition

Reply:
Then `--if-installed` as it filters out not-installed files? This is my last try :-)

Reply:
OK, that's a good solution. Done in fafaaf25bc